web/beacon-app/src/features/home/components/WelcomeAttention.tsx

```diff
@@ -22,14 +22,23 @@ const WelcomeAttention = () => {
       >
         <p className="text-md">
           <Trans>
-            Welcome to Ensign! Get started on your first project. We’ll guide you along the way!
+            Welcome to Ensign! Set up or manage your projects. A project is{' '}
+            <a
+              href={LINK}
+              target="_blank"
+              rel="noreferrer"
+              className="font-bold text-[#1D65A6] hover:!underline"
+            >
+              a database for events.
+            </a>{' '}
+            We’ll guide you along the way!
           </Trans>
         </p>
 
         <Button
-          //variant="ghost"
+          variant="ghost"
           size="small"
-          className="!bg-green text-white hover:!bg-green/[0.8]"
+          className="border-none bg-[#37A36E] text-white hover:!bg-[#37A36E]/[0.8]"
```

Comment:
The button doesn't appear when I view the changes locally. I have to hover over where the button should be for it to appear. This continued after I ran `yarn install`.  https://www.awesomescreenshot.com/image/39966728?key=d922e6587c052fb7844b3f375b70e735